package.json

```
@@ -8,8 +8,8 @@
     "url": "git@github.com:blablacar/ui-library.git"
   },
   "engines": {
-    "node": "^12.16.1",
-    "npm": "^6.13.4"
+    "node": "^15.11.0",
+    "npm": "^7.6.0"
```

```suggestion
    "node": "^12.16.1 | ^15.11.0",
    "npm": "^6.13.4 | ^7.6.0"
```

Maybe this to be backward compatible?

---

Trying this syntax but it doesn't seems to work. 
I got the following working:

```
"node": ">=12.16.1 <=v15.11.0",
"npm": ">=6.13.4 <=7.6.0"
```

WDYT?

---

👌

---

You're right. Moreover, the `.nvmrc` ensure the version so we're good IMHO.